src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,138 @@
+
+## File Naming Conventions
+
+We use two letters prefixes to name our files, this two letters depending on the pattern type and look like that: **_[XX] File name.sketch_**
+
+| Prefix   | Pattern type        | Example                                     |
+| -------- | ------------------- | ------------------------------------------- |
+| **[AA]** | Styles              | Colors, icons, fonts, shadows, ...          |
+| **[BA]** | Simple Components   | Buttons, links, tags, forms, ...            |
+| **[BB]** | Complex Components  | Notifications, accordions, popins, tabs ... |
+| **[ZZ]** | Specific librairies | Tools, documentation, ...                   |
+
+## Sketch
+
+Please follow these rules to have the same architecture and standards in every library. This will be very helpful for the design teams to understand every file in Mozaic.
+
+### Symbol categorisation
+
+We use this categorisation to create consistency in our libraries:
+
+| Categrory             | Usage                                                                                                                                                    |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **👾 For developers** | These symbols should not be used in your designs. They are meant to share the different states of a component with the development team.                 |
+| **🛠 ToolKit**         | This section contains the symbols used to build the final symbols. They are not meant to be used directly but to be accessible through symbol overrides. |
+| **Component**         | This is where you'll find the ready to use symbols of the library.                                                                                       |
+
+![buttons--sketch-menu.png](buttons--sketch-menu.png)
+
+<br />
+
+**Tips to create hierarchy:**
+
+- We use numbers as prefixes to order components depending on their importance. For example, `Button / 01-Solid / 01-S / 01-Primary`
+
+![hierarchy--importance](hierarchy--importance.png)
+
+- We use a double hyphen as suffixes to indicate a specific information. For example, `Button / 01-Solid / 01-S / 01-Primary--Campus`
+
+![hierarchy--specific](hierarchy--specific.png)
+
+- We use clothes sizes letters to organize the patterns from the smallest to the biggest. For example, `XS`, `S`, `M`, `L`,`XL`.
+
+![sizes-naming](sizes-naming.png)
+
+### Layers naming
+
+#### Shapes
+
+Please use the following naming conventions for your shapes:
+
+| Type           | Description                                                                  |
+| -------------- | ---------------------------------------------------------------------------- |
+| **Frame**      | Frame is the combination of a **background**, an **outline** or a **shadow** |
```

Comment:
On `figma` frame is the artboard. It might be confusing. Unless it's the standards on design (i don't remember anymore, 😮 )